src/System.Management.Automation/DscSupport/ConvertCimMofToJsonCommand.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.PowerShell.DesiredStateConfiguration.Internal
+{
+    /// <summary>
+    /// Convert-CimMofToJson cmdlet implementation
+    /// </summary>
+    [Cmdlet(VerbsData.Convert, "CimMofToJson")]
+    public sealed class ConvertCimMofToJsonCommand : Cmdlet
+    {
+        /// <summary>
+        /// Top level directory to serach for .mof files
+        /// </summary>
+        /// <value>Test</value>
+        [Parameter(ValueFromPipeline = true, Position = 0)]
+        [ValidateNotNullOrEmpty]
+        public string Directory { get; set; }
```

Comment:
```suggestion
        public string[] Directory { get; set; }
```

Reply:
I think it is complicate cmdlet without need - we can get `dir` output or directory list from pipeline as attribute states.

Reply:
The parameter already accepts pipeline input. Would it be that much more work to support arrays?

Reply:
Changed parameter to be string array and added supporting code.
Change was done in `PSDesiredStateConfiguration` module because `ConvertTo-DscJsonSchema` cmdlet was moved there.